### PR TITLE
Parse parameters from a file

### PR DESCRIPTION
**Pull request by tjovicic**
*Issue #, if available:* 

*Description of changes:* Read parameters from a file

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -33,3 +35,25 @@ export function parseString(s: string): string | undefined {
 export function parseNumber(s: string): number | undefined {
   return parseInt(s) || undefined
 }
+
+export function parseParameters(parameterOverrides: string): Parameter[] {
+  if (parameterOverrides.trim().startsWith('file://')) {
```
Comment (katallaxie) [inline src/utils.ts:40]: What do you think of doing this using the node built in https://nodejs.org/api/url.html and https://nodejs.org/api/fs.html of which `readFileSync` does support `file://`. We could then think also of loading from a `http` url.

Comment (tjovicic) [reply src/utils.ts:40]: aa nice, i'll check it out!

File: src/utils.ts [hunk 2]
```diff
@@ -33,3 +35,25 @@ export function parseString(s: string): string | undefined {
...
+    const path = new URL(parameterOverrides.trim())
```
Comment (tjovicic) [inline src/utils.ts:41]: @katallaxie Something like this?

**Comment by tjovicic** [comment]
> You need to adopt the docs in `action.yml` to reflect the change.

thanks, i forgot :)

**Pull request merged**